README.md

```diff
@@ -55,6 +61,114 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+## Adding remote repositories
+
+Configure the registry viewer through environment variables.
+
+- `DEVFILE_SOURCE_REPOS` - Source repository(ies) name
+- `DEVFILE_ENDPOINTS` - Endpoint(s) that serves the devfile JSON
+- `DEVFILE_DIRECTORIES` - Endpoint(s) that serves the devfile
+
+Notes about environment variables:
+
+- The environment variables DEVFILE_SOURCE_REPOS, DEVFILE_ENDPOINTS, and DEVFILE_DIRECTORIES must be the same array length and in order.
+- Multiple sources should be split by ",".
+
+```
+DEVFILE_SOURCE_REPOS="Official"
+DEVFILE_ENDPOINTS="https://registry.devfile.io/index/all?icon=base64"
+DEVFILE_DIRECTORIES="https://registry.devfile.io/devfiles/"
+```
+
+Configure the registry viewer through config file.
+
+- `/config/devfile-endpoints.json` - Endpoint(s) that serves the devfile JSON
+- `/config/devfile-directories.json` - Endpoint(s) that serves the devfile
+
+`/config/devfile-endpoints.json`
```

Comment:
Like my other comment re: env vars, I think it'd be more convenient to have one single config file like so:
```
{
  "Official": "https://registry.devfile.io/",
  "Product": "https://registry.redhat.com"
}
```

And then where needed, have the code append the appropriate endpoint in the API calls. Thoughts?

Reply:
This is currently being changed to a single config file `config/devfile-registry-hosts.json` with the data looking like this. 
``` json
{
  "Test1": {
    "url": "https://registry.devfile.io"
  },
  "Test2": {
    "index": "/config/examples.json",
    "stacks": "/stacks"
  }
}
```

Reply:
this looks nice

Reply:
👍

Reply:
Fixed